core/postsolve.gms

```diff
@@ -54,7 +54,7 @@ display s_actualbudgetco2;
 );
 
 if(cm_iterative_target_adj eq 5,
-*JeS* Update tax levels/ multigasbudget values to reach the CO2 budget (s_actualbudgetco2 runs from 2020-2100)
+*JeS* Update CO2 tax trajectories / cumulative emission level to reach the budget target (s_actualbudgetco2 runs from 2020-2100, not peak budget)
```

Comment:
would stick with the same wording for all realizations, i.e. tax levels / budgets